test/java/seedu/addressbook/commands/UpdateCommandTest.java

```diff
@@ -0,0 +1,157 @@
+package seedu.addressbook.commands;
+
+import org.junit.Before;
+import org.junit.Test;
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.ui.TextUi;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
```

Comment:
What is wrong with such import statements? (hint: refer to the coding standards :P)

Reply:
We're not supposed to use * in import statements